### PR TITLE
Close gap in vocab for AutoTokenizer support for GPT4Tokenizer

### PR DESCRIPTION
**Pull request by sayanshaw24**
For WSSI work as requested by Ran.

### CODE REVIEW

**Review by RandySheriffH — APPROVED** [review]

File: onnxruntime_extensions/_hf_cvt.py [hunk 1]
```diff
@@ -23,12 +25,31 @@ def __init__(self, tokenizer):
...
     def bpe_tokenizer(self, **kwargs):
         hf_gpt2_tokenizer = self.tokenizer
+        attrs = None
 
         if type(self.tokenizer).__name__.endswith('Fast'):
             raise ValueError('Please use the slow version of the tokenizer (ex: GPT2Tokenizer).')
-
-        attrs = {'vocab': json.dumps(
-            hf_gpt2_tokenizer.encoder, separators=(',', ':'))}
+        elif(self.tokenizer.name_or_path.endswith('gpt-4')):
```
Comment (natke) [inline onnxruntime_extensions/_hf_cvt.py:32]: Can we add a comment here?

File: test/test_autotokenizer.py [hunk 2]
```diff
@@ -68,6 +68,17 @@ def test_clip_tokenizer(self):
             pre_kwargs={"WITH_DEFAULT_INPUTS": True})[0])
         actual_ids = ort_tok([text])[0]
         np.testing.assert_array_equal(ids, actual_ids)
+        
+    def test_gpt2_tokenizer(self):
```
Comment (natke) [inline test/test_autotokenizer.py:72]: Does the GPT-4 tokenizer get tested by this code?

File: onnxruntime_extensions/_hf_cvt.py (hunk 1 above)
Comment (natke) [reply onnxruntime_extensions/_hf_cvt.py:32]: to explain why we need the if clause

Comment (sayanshaw24) [reply onnxruntime_extensions/_hf_cvt.py:32]: yep, i added one in L33!

File: test/test_autotokenizer.py (hunk 2 above)
Comment (sayanshaw24) [reply test/test_autotokenizer.py:72]: yep - to be more precise though, the gpt4 vocab gets tested - the actual tokenization uses the same alg as the gpt2 tok which is why loading it from HF uses the HF `GPT2Tokenizer` class as well.

**Pull request merged**